<launch>
    <param name="robot_description" command="$(find xacro)/xacro --inorder $(find sawyer_description)/urdf/sawyer.urdf.xacro electric_gripper:=true" />

    <rosparam
        file="$(find sawyer_kdl)/config/saywer_kdl_params.yaml"
        command="load" />

    <rosparam
        file="$(find sawyer_description)/params/named_poses.yaml"
        command="load" />

    <rosparam command="load" file="$(find sawyer_move)/config/yoyo_param.yaml" />

    <rosparam command="load" file="$(find sawyer_move)/config/tracking_param.yaml" />

    <node name="sawyer" pkg="sawyer_move" type="force_control" output="screen" cwd="node" >
      <remap from="joint_states" to="robot/joint_states"/> 
    </node>
    
    <node name="flir_tracking_test" pkg="sawyer_move" type="flir_tracking_test" output="screen" cwd="node" />
    <node name="tracking" pkg="sawyer_move" type="tracking" output="screen" cwd="node" />
    <node name="eotracking" pkg="sawyer_move" type="eotracking" output="screen" cwd="node" />
    <!-- <node name="flirtracking" pkg="sawyer_move" type="flirtracking" output="screen" cwd="node" /> -->
    <node name="collect_data" pkg="sawyer_move" type="collect_data" output="screen" cwd="node" />
    <node name="klecontrol" pkg="sawyer_move" type="klecontrol" output="screen" cwd="node" />
    <!-- <node name="control" pkg="sawyer_move" type="control" output="screen" cwd="node" /> -->
</launch>
